arduino-ide-extension/src/browser/arduino-preferences.ts

```diff
@@ -64,13 +64,32 @@ export const ArduinoConfigSchema: PreferenceSchema = {
       ),
       default: 0,
     },
+    /*
     'arduino.ide.autoUpdate': {
       type: 'boolean',
+      default: true,
       description: nls.localize(
         'arduino/preferences/ide.autoUpdate',
-        'True to enable automatic update checks. The IDE will check for updates automatically and periodically.'
+        'True to enable automatic updates'
       ),
+    },
+    'arduino.ide.checkForUpdatesOnStartup': {
+      type: 'boolean',
       default: true,
+      description: nls.localize(
+        'arduino/preferences/ide.checkForUpdatesOnStartup',
+        'Check for IDE updates on start-up'
+      ),
+    },
+    */
+    'arduino.ide.updateChannel': {
+      type: 'string',
+      enum: ['latest', 'nightly', 'none'],
+      default: 'none',
```

Comment:
`none` means the IDE won't check for updates?

Reply:
Right now the channel is hard-coded, but I'm about to make a change to make use of this setting to know which channel to use when checking for updates.

I think I'll change this code like this:
```suggestion
      enum: ['latest', 'nightly'],
      default: 'latest',
```
so that by default the user will check for the latest release. I'd remove `none` because it doesn't make any sense to me. Do you agree?

Reply:
Yep, that's perfect.